Vegeting/Global/UIComponent/InterestView.swift

```diff
@@ -7,28 +7,44 @@
 
 import UIKit
 
+protocol InterestViewDelegate: AnyObject {
+    func setBottomButtonEnabled(selectedList: [String])
+}
+
 class InterestView: UIView {
     
     // MARK: - properties
     
+    enum EntryPoint {
+        case profile
+        case register
+    }
+    
     private let interestCollectionView: UICollectionView = {
         let layout = UICollectionViewFlowLayout()
-        layout.scrollDirection = .horizontal
-        layout.minimumInteritemSpacing = 16
+        layout.minimumInteritemSpacing = 12
+        layout.minimumLineSpacing = 16
         
         let collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
```

Comment:
p1;
밑과 같이 allowMultipleSelection이라는 변수를 true로 바꿔주게 되면 collectionView에서도 다수의 것들을 선택할 수 있게 됩니다!
```suggestion
        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
        collectionView.allowsMultipleSelection = true
```

Reply:
제가 isSelectedCell을 이용해서 구현했던 이유가 여러개가 선택이 안되었기 때문인데 요런게 있었네요!! 이게 있으면 굳이 사용할 필요가 없는 것 같습니다! `allowsMultipleSelection` 좋은 키워드 배워갑니다!!

Reply:
https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Vegeting/pull/143/commits/f25642e9e854c1afa40d2ecf8461b3c42ed4e218#diff-6a6bccd0ca2b0fd81a525b987ade6ff6fce7b9f9db21ec2cdeba4f6f75abe205R44
에서 반영완료~!